<launch>

    <node pkg="sensors" type="cu_cr_2.py" name="cu_cr_2" />

    <node name="asmc_m2" pkg="sensors" type="asmc_m2" >
	<param name = "k_u" value = "0.1269" />
	<param name = "k_psi" value = "0.2" />
	<param name = "kmin_u" value = "0.01" />
	<param name = "kmin_psi" value = "0.01" />
	<param name = "k2_u" value = "0.01" />
	<param name = "k2_psi" value = "0.01" />
	<param name = "mu_u" value = "0.1" />
	<param name = "mu_psi" value = "0.1" />
	<param name = "lambda_u" value = "0.001" />
	<param name = "lambda_psi" value = "1" />
	<param name = "alpha_u" value = "0.33" />
	<param name = "alpha_psi" value = "0.2468" />
	<param name = "L1_u" value = "0" />
	<param name = "L1_psi" value = "0" />
    </node>

    <!--Launch rosbag to record topics-->
    <node type="record" pkg="rosbag" name="recorder" args="record -a -o /home/ubuntu/catkin_ws/src/sensors/bags/asmc/ftc2_test" />

</launch>
